apps/antalmanac/tests/calendarize-helpers.test.ts

```diff
@@ -0,0 +1,227 @@
+import { describe, test, expect } from 'vitest';
```

Comment:
Massive, MASSIVE W for including unit tests in your PR. This is a historic moment for AntAlmanac. But, why use vitest? It describes itself as being native to Vite. I mean, obviously if we're able to use it here then it works, but why not use a framework made for React instead? This might bite us in the ass later if we want to test components instead of pure functions.

Reply:
What? This concern literally makes no sense lol. Jest, Mocha, Jasmin, Vitest, etc. are all testing frameworks. They're not responsible for testing UI by themselves. You setup integrations and stuff with the framework of your choice. i.e. here's an example of react-testing-library https://www.robinwieruch.de/vitest-react-testing-library/

TLDR: this is like saying "but Vite is targeted for first-class support for Vue, so it might not be the best choice for React"

Reply:
Yeah, I agree it made no sense. Embarrassing moment there 😳

Reply:
Imagine confusing names between JS frameworks. I can't imagine not knowing the difference between React and Preact, or Svelte and SvelteKit, or Vuepress and Vitepress. :pensive:

Reply:
> Massive, MASSIVE W for including unit tests in your PR. This is a historic moment for AntAlmanac. For a second I was having a hang-up on why we're using vitest because I was confusing Vue and Vite 🤦‍♂️🤦‍♂️🤦‍♂️. Anyways, if you read that before I edited this just ignore it.
> 
> Edit: I stand corrected, just realized you had already merged in a test before this PR. How did nobody make a big deal out of that or ask about it in the review for that PR??

Oh, and it's because I added the tests and reviewed the same PR lol